doc/api/net.markdown

```diff
@@ -51,7 +51,8 @@ var server = net.createServer((socket) => {
 });
 
 // grab a random port.
-server.listen(() => {
+server.listen((err) => {
+  if(err) throw err;
```

Comment:
Can you add a space between `if` and `(`. And preferably put `throw err;` on a separate line. Same comment on the other changes.